resources/views/dashboard/controllers/ticket.blade.php

```diff
@@ -39,7 +39,7 @@
             <p><b>Movements:</b> @if($ticket->movements) {{ $ticket->movements }} @else N/A @endif</p>
             <p><b>Comments:</b></p>
             @if($ticket->comments != null)
-                <p>{!! nl2br(e($ticket->comments)) !!}</p>
+                <p>{!! nl2br($ticket->comments) !!}</p>
```

Comment:
@kjporter is `nl2br()` actually necessary? I feel like I tried not using it elsewhere and it worked - if it works without, we should start removing these to reduce clutter